spanner/system-test/spanner.test.js

```diff
@@ -53,103 +53,139 @@ test.after.always(async (t) => {
 
 // create_database
 test.serial(`should create an example database`, async (t) => {
-  const output = await tools.runAsync(`${schemaCmd} createDatabase "${INSTANCE_ID}" "${DATABASE_ID}"`, cwd);
-  t.true(output.includes(`Waiting for operation on ${DATABASE_ID} to complete...`));
-  t.true(output.includes(`Created database ${DATABASE_ID} on instance ${INSTANCE_ID}.`));
+  const results = await tools.runAsyncWithIO(`${schemaCmd} createDatabase "${INSTANCE_ID}" "${DATABASE_ID}"`, cwd);
```

Comment:
What's the point of this?

`stderr` should be ignored unless we expect an error to occur. (If an error occurs when it shouldn't, our test _should_ fail.)

Reply:
I've discover lately that sometimes tests/commands fail silently (return a 0 exit code), but _do_ print some useful info to stderr. This improves test debuggability.